File: src/lib/error-test.ts
/**
 * This file contains intentional errors for CI failure testing
 * DO NOT MERGE
 */

// TypeScript error: using unknown variable
export function brokenFunction() {
	return unknownVariable + 123;
}

// Biome format error: inconsistent formatting
export const badFormat={value:1,name:"test",data:[1,2,3]};

// Unused variable (lint error)
const unusedVariable = "This will trigger lint error";

File: src-tauri/contexts/shared/src/error_test.rs
// This file contains intentional errors for CI failure testing
// DO NOT MERGE

// Format error: bad formatting
pub fn badly_formatted(){let x=1;let y=2;return x+y;}

// Clippy error: needless return
pub fn needless_return_example() -> i32 {
    return 42;
}

// Clippy error: unused variable
pub fn unused_var() {
    let _unused = 123;
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test_that_will_fail() {
        // This test will intentionally fail
        assert_eq!(1 + 1, 3, "Math is broken!");
    }
}

